libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -1114,6 +1114,30 @@ bool AP_AHRS_NavEKF::healthy(void) const
     return AP_AHRS_DCM::healthy();
 }
 
+bool AP_AHRS_NavEKF::prearm_healthy(void) const
+{
+    bool prearm_health = false;
+    switch (ekf_type()) {
+    case 0:
+        prearm_health = true;
+        break;
+
+    case 2:
+    default:
+        if (_ekf2_started && EKF2.all_cores_healthy()) {
+            prearm_health = true;
+        }
+        break;
+
+    case 3:
```

Comment:
Don't we need a `case 10`?  Better still, an enumeration so we don't miss cases?

Reply:
argh, yes I forgot `case EKF_TYPE_SITL` when I yanked lines :)

Reply:
Fixed. I yanked most of the switch down which is where the numbers came from in the first place :(